docs/plugins/overview.md

```diff
@@ -38,11 +38,25 @@ Where:
 
 ## Usage {#usage}
 
-A plugin has two usages:
+Plugins generally can have three usages:
 
+- [Utility plugin](#utility-plugin): Load them in your `init.lua`.
```

Comment:
I think https://yazi-rs.github.io/docs/plugins/overview#sync-vs-async already included this, i.e. `require`-ing a plugin

Reply:
My confusion was that I thought that plugins can only either do actions on a keypress or as previewers/preloaders, with no other possibility of integration. How can we word this better?